msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -42,6 +42,11 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public sealed partial class PublicClientApplication : ClientApplicationBase, IPublicClientApplication
     {
+        static PublicClientApplication()
+        {
+            ModuleInitializer.EnsureModuleInitialized();
```

Comment:
Why not in ClientApplicationBase rather?

Reply:
Static constructors do not fire unless the actual class is used. It does not fire if you use a derived class.

Reply:
oh, I did not know! thanks for the explanation.